src/status_im2/contexts/communities/overview/view.cljs

```diff
@@ -12,23 +12,8 @@
             [status-im2.contexts.communities.overview.style :as style]
             [status-im2.contexts.communities.actions.community-options.view :as options]
             [quo2.components.navigation.floating-shell-button :as floating-shell-button]
-            [status-im2.contexts.communities.overview.utils :as utils]
```

Comment:
I left a comment in the issue to restore this feature to use this code 👍 

https://github.com/status-im/status-mobile/issues/15961